conf/ui.yaml.template

```
@@ -24,7 +24,8 @@ UI:
   WEBDRIVER: chrome
   # Binary location for selected wedriver (not needed if using saucelabs)
   WEBDRIVER_BINARY: /usr/bin/chromedriver
-
+  RECORD_VIDEO: false
+  GRID_URL: ""
```

Thanks for adding this settings here, just suggestion to add an example URL here instead
```suggestion
  GRID_URL: http://infra-grid.example.com:4444
```

---

Also, add required `se:recordVideo: '@jinja {{ this.ui.record_video }}'` in webdriver options in desired_capabilities

---

Done